File: CS50x/Problem Sets/Week 6/mario-less.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break
for i in range(0, h):
    for j in range(0, h - i - 1):
        print(" ", end="")
    for j in range(0, i + 1):
        print("#", end="")
    print()

File: Problem Sets/Week 1/mario-less.c
#include <cs50.h>
#include <stdio.h>

void pattern(int n)
{
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

int main()
{
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1 || n > 20);
    {
        pattern(n);
    }
}
